anitya/db/models.py

```diff
@@ -345,6 +346,11 @@ class Project(Base):
     latest_version = sa.Column(sa.String(50))
     logs = sa.Column(sa.Text)
 
+    last_check = sa.Column(sa.TIMESTAMP(timezone=True),
+                           default=arrow.utcnow().datetime, index=True)
+    next_check = sa.Column(sa.TIMESTAMP(timezone=True),
+                           default=arrow.utcnow().datetime, index=True)
```

Comment:
This probably isn't what you want. The default here is a scalar, not a function. ``default`` accepts both, but what I expect you want to happen is for whatever the current time is when this project is added. This will be whatever time the code first runs (probably import time). You could do ``lambda: arrow.utcnow().datetime``.

Reply:
Thanks for note, didn't noticed this.